docs/user-guide/federation/index.md

```diff
@@ -46,3 +46,13 @@ The following guides explain some of the resources in detail:
 
 [API reference docs](/federation/docs/api-reference/readme/) lists all the
 resources supported by federation apiserver.
+
+## Cascading deletion
+
+kubernetes v1.5 includes alpha support of DeleteOptions.OrphanDependents for
+federated resources. Ingresses, Namespaces, Replica Sets, Secrets, Deployments
+and Daemon Sets support it. This means that when any of these resources are
+deleted from federation control plane with DeleteOptions.OrphanDependents =
+false, then the corresponding resources are also deleted from all underlying
+clusters. By default, deleting the resource from federation control plane does
+not delete the corresponding resources from underlying clusters.
```

Comment:
There are already two the's in the sentence. I dont think we need another one here.